### DISCUSSION TITLE
Split `renovate-config-validator` as separate package

### DISCUSSION THREAD

**Discussion by avaly** [renovatebot/renovate#14418] [Suggest an Idea]
### What would you like Renovate to be able to do?

I would like to setup a CI test for our Renovate config file using the `renovate-config-validator` binary.

However, installing the NPM `renovate` package brings in a **huge** list of dependencies ([580 transitive packages](https://npm.anvaka.com/#/view/2d/renovate)), which most likely are not used by the validator script.

### If you have any ideas on how this should be implemented, please tell us here.

Publish the `renovate-config-validator` as a standalone NPM package.

### Is this a feature you are interested in implementing yourself?

No

**Comment by rarkins** [comment]
Splitting it isn't going to achieve what you want, because any standalone validator would need to require `renovate` as a dependency, and therefore install the same dependencies. It only work if we did some type of compiled, tree-shaken publish.
